config/locales/en/course/user_invitations.yml

```diff
@@ -33,7 +33,8 @@ en:
           individual: 'Individually add users'
           registration_code: 'Registration code'
         file:
-          format: 'Files uploaded must be in the following format:'
+          format: 'Files uploaded must be in the following format (Roles are optional):'
```

Comment:
Suggest to combine the role_options translation with this. Also explain that when the role is not specified, the default role (student) will be selected.

Reply:
Alright, will fix.